crates/worker/src/partition/action_effect_handler.rs

```diff
@@ -96,6 +100,25 @@ impl ActionEffectHandler {
                     .await?;
                 }
             }
+            ActionEffect::ScheduleCleanupTimer(invocation_id, duration) => {
```

Comment:
Maybe add a comment explaining why we need to propose and agree on the clean-up timer.